Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -251,6 +263,11 @@ def network_create_engine_wrapper(self, pstate, initialdlstatus=None):
             atp["duplicate_is_error"] = True
             atp["hops"] = self.get_hops()
 
+            if share_mode:
+                atp["flags"] = lt.add_torrent_params_flags_t.flag_share_mode
+
+            self.set_checkpoint_disabled(checkpoint_disabled)
```

Comment:
why is `self.set_checkpoint_disabled` called twice (in `setup` and `network_create_engine_wrapper`)?